hack/update/update.go

```diff
@@ -93,11 +93,13 @@ func Apply(schema map[string]Item, data interface{}) {
 	changed, err := fsUpdate(FSRoot, schema, data)
 	if err != nil {
 		klog.Errorf("Unable to update local repo: %v", err)
+		return err
```

Comment:
If we return an error, why log the error here? This will lead to double logging for the same error which makes it much harder to debug issues.

We can do:

    return fmt.Errorf("Unable to update local repo: %w", err")